tools/sof-fw-boot-test.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+
+"""Module to stress test firmware boot"""
+
+import argparse
+import logging
+import sys
+import time
+
+# define command line arguments
+def parse_cmdline():
+	"""Function to parse the command line arguments"""
+	parser = argparse.ArgumentParser(add_help=True, formatter_class=argparse.RawTextHelpFormatter,
+									 description='A script for stress testing firmware boot')
+	parser.add_argument('-i', '--iter', type=int, default=100,
+						help="number of firmware boot iterations")
+	parser.add_argument('-f', '--firmware', type=str, required=True, help='firmware filename')
+	parser.add_argument('-p', '--fw_path', type=str, required=True,
+						help='path to the firmware file relative to /lib/firmware')
+	return vars(parser.parse_args())
+
+def main():
+	"""Main function for stress testing"""
+	logging.basicConfig(level=logging.INFO, format='%(message)s')
```

Comment:
Move this to the top-level so:

- other functions can use it
- interactive use does not crash

Reply:
done